src/Google/AdsApi/AdWords/Testing/BatchJobs/v201609/BatchJobsTestProvider.php

```diff
@@ -100,6 +100,15 @@ public static function getMutateResultArray() {
     $mutateResult2->setResult($operand);
     $mutateResult2->setIndex(1);
 
-    return [$mutateResult1, $mutateResult2];
+    $campaign = new FakeCampaign();
+    $campaign->setId(9223372036854775900);
+    $campaign->setName('Test large ID');
+    $operand = new FakeOperand();
+    $operand->setCampaign($campaign);
```

Comment:
Never mind. I can do that when I sync this with our codebase.
Thanks for your contribution!